exercises/practice/rna-transcription/.meta/config.json

```diff
@@ -1,6 +1,30 @@
 {
   "blurb": "Given a DNA strand, return its RNA Complement Transcription.",
-  "authors": [],
+  "authors": [
+    "kytrinyx"
+  ],
+  "contributors": [
+    "alxndr",
+    "budmc29",
+    "cadwallion",
+    "Cohen-Carlisle",
+    "connorlay",
+    "dkinzer",
+    "hilary",
+    "iHiD",
+    "Insti",
+    "jmay",
+    "jpotts244",
+    "kickinbahk",
+    "kotp",
+    "mambocab",
+    "mhelmetag",
+    "mike-hewitson",
+    "NobbZ",
```

Comment:
I can not remember having contributed to this exercise, please remove me from this list.

Reply:
Github remembers: https://github.com/exercism/ruby/pull/175